### DISCUSSION TITLE
More options for the reminder

### DISCUSSION THREAD

**Discussion by tothpsk** [streetwriters/notesnook#2033] [Ideas]
Hi, it would be nice to have more options when reminding some tasks. For example, having the option to set up a yearly reminder or a custom reminder (after xxx days/months/years, after 5 repetitions...).
Thanks for consideration,

Peter

**Comment by livejamie** [comment]
Agreed. The reminder feature is strange right now. It needs to be clarified what the difference between reminders and tasks is. I assume they are the same system. The app treats them differently.

Once a reminder happens, there's no way to take action; I can't check it off. I don't even know how to delete it.
